test/uploads/config.ts

```diff
@@ -291,6 +291,7 @@ export default buildConfigWithDefaults({
     {
       slug: mediaSlug,
       fields: [],
+      disableDuplicate: false,
```

Comment:
I would remove `disableDuplicate` in this test config so that we are testing that it works by *default*.